src/qibojit/backends/gpu.py

```diff
@@ -19,9 +19,18 @@ def __init__(self):
 
         import cupy as cp  # pylint: disable=import-error
         import cupy_backends  # pylint: disable=import-error
+        from qibo import __version__ as qibo_version
+
+        from qibojit import __version__ as qibojit_version
 
         self.name = "qibojit"
         self.platform = "cupy"
+        self.versions = {
+            "qibo": qibo_version,
+            "qibojit": qibojit_version,
+            "numpy": self.np.__version__,
+            "cupy": self.cp.__version__,
```

Comment:
```suggestion
            "cupy": cp.__version__,
```
or move this under the `self.cp = cp` definition. This should fix the pylint issue.

Reply:
Oh yes, sorry. I didn't realize that tests were failing. Thanks!